app/views/pages/events/registration/step2.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="event-reg-main">
+    <div class="event-reg-main__content container-1000">
+        <div class="event-reg-main__content__left">
+            <h1>Sign up for this event:</h1>
+            <h2>Train to Teach Coventry online event</h2>
+            <p></p>
+
+            <div class="govuk-form-group">
+                <div class="fieldlabel">Phone number (optional)</div>
+                <p class="fieldhelp">This helps us contact you with updates to this event and other important information.</p>
+                <input class="govuk-input" id="phone-number" name="phone-number" type="text">
```

Comment:
May be worth checking if you can use specific inputs, like input type `tel` here as it'll have a nicer experience especially on mobile it should show the number pad.

Reply:
ill make a note and check the gov style docs